src/module_MEDIATOR.F90

```diff
@@ -4892,6 +4899,14 @@ subroutine MedPhase_atm_ocn_flux(gcomp, rc)
     character(ESMF_MAXSTR) ,pointer  :: fieldNameList(:)
     real(ESMF_KIND_R8), pointer :: zbot(:,:),ubot(:,:),vbot(:,:),thbot(:,:), &
                                    qbot(:,:),rbot(:,:),tbot(:,:), pbot(:,:)
+! LHC 2020
```

Comment:
Remove LHC 2020 (and all other places it appears).

Reply:
removed in the "updated" develop branch